prov/util/src/util_ns.c

```diff
@@ -179,7 +179,8 @@ static int util_ns_map_lookup(struct util_ns *ns, void *service_in,
 static void util_ns_name_server_cleanup(void *args)
 {
 	void **cleanup_args = (void **)args;
-	ofi_close_socket((uintptr_t)cleanup_args[0]);
+	ofi_close_socket((uintptr_t)*((SOCKET *)cleanup_args[0]));
```

Comment:
I think this should be cast to *((SOCKET *) -- without the uintptr_t.

Reply:
Sure. Done
Thanks!